testing/kuttl/scripts/postgres-version-check.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+EXPECTED_VERSION=$1
+
+CLUSTER=${CLUSTER:-default}
+
+POSTGRES_POD=$(kubectl -n ${NAMESPACE} get pod --selector=postgres-operator.crunchydata.com/cluster=${CLUSTER},postgres-operator.crunchydata.com/instance-set=instance1 --no-headers -o custom-columns=":metadata.name")
+
+# Grab the Postgres major version from `version()` output to compare to the provided value
+VERSION=$(kubectl -n ${NAMESPACE} exec -i ${POSTGRES_POD} -c database -- psql -c "SELECT version();" |
+    grep PostgreSQL |
```

Comment:
```suggestion
VERSION=$(kubectl -n ${NAMESPACE} exec -i ${POSTGRES_POD} -c database -- psql -tc "SELECT version();" |
```
`-t` flag will drop the header of the response (tuple only)

Reply:
:+1: Works for me, updated.